components/schema-migrator/migrations/1_initialize_schema.up.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TYPE runtime_status_condition AS ENUM ('INITIAL', 'READY', 'FAILED');
+
+CREATE TABLE runtime
```

Comment:
I have found a recommendation that table names should be pluralized: https://www.periscopedata.com/blog/better-sql-schema